blocks/article-header/article-header.js

```diff
@@ -85,22 +86,22 @@ function buildSharing() {
   const sharing = document.createElement('div');
   sharing.classList.add('article-byline-sharing');
   sharing.innerHTML = `<span>
-      <a data-type="Twitter" data-href="https://www.twitter.com/share?&url=${url}&text=${title}">
+      <a data-type="Twitter" data-href="https://www.twitter.com/share?&url=${url}&text=${title}" alt="Share post on Twitter" aria-label="Share post on Twitter">
         ${createSVG('twitter').outerHTML}
       </a>
     </span>
     <span>
-      <a data-type="LinkedIn" data-href="https://www.linkedin.com/shareArticle?mini=true&url=${url}&title=${title}&summary=${description || ''}">
+      <a data-type="LinkedIn" data-href="https://www.linkedin.com/shareArticle?mini=true&url=${url}&title=${title}&summary=${description || ''}" aria-label="Share post on LinkedIn" alt="Share post on LinkedIn">
```

Comment:
For labels like this, you can update the placeholders.xlsx in each locale on sharepoint for translation.

Reply:
I ended up using Google Translate and updated the placeholders.xlsx for all languages (br, de, es, fr, it, jp and ko) for blog.adobe.com.